src/genie/libs/parser/iosxe/tests/ShowCtsRoleBasedPermissions/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,97 @@
+expected_output = {
```

Comment:
I believe this number is index. if so, better to have key name as `indexes` like below:
```
"indexes": {
    1: {
        "policy_name": "default",
        (snip)

Reply:
Updated